common/include/fwd-window.h

```diff
@@ -36,18 +36,18 @@ int window_close(window *wind);
 int window_exists(window *wind);
 window *window_get_front();
 window *window_get_first();
-window *window_get_next(window &wind);
-window *window_get_prev(window &wind);
+static inline window *window_get_next(window &wind);
```

Comment:
OK, I was unsure if the compiler would still inline them. Removing the `static inline` keywords but leaving the function declarations (except for `window_is_visible`, which has been replaced with the `is_visible` method), otherwise I found the friend functions are declared within the `dcx::window` namespace.